docs/howto/mock_services.md

```diff
@@ -246,27 +246,21 @@ object AccountObserverSpec extends DefaultRunnableSpec {
 
 ## Mocking multiple collaborators
 
-In some cases we have more than one collaborating service being called. In such situations we need to build our expectations separately for each
-service and then combine them into single environment:
+In some cases we have more than one collaborating service being called. You can create mocks for rich environments, a mockable for it will be automatically derrived if you have implicit mockables for each part of composed environment in scope.
```

Comment:
```suggestion
In some cases we have more than one collaborating service being called. You can create mocks for rich environments, a mockable for it will be automatically derived if you have implicit mockables for each part of composed environment in scope.
```

Reply:
Yes, fixing failing CI tests one by one now.